sqe-api-server/Services/ImageService.cs

```diff
@@ -13,6 +14,7 @@ public interface IImageService
     {
         ImageDTO ImageToDTO(Image model);
         Task<ImageInstitutionListDTO> GetImageInstitutionsAsync();
+        Task<InstitutionalImageListDTO> GetInstitutionImagesAsync(string institution);
```

Comment:
Don't we need an institution ID and an institution DTO with maybe more than just a name, and some institution metadata?

Reply:
I'll make a note and see what other metadata we might want to add for the institution. It might be nice to have a little more info.